pkg/webui/containers/fetch-table/index.js

```diff
@@ -304,6 +310,8 @@ const FetchTable = props => {
           page={page}
           totalCount={totalCount}
           pageSize={pageSize}
+          setPageSize={setPageSize}
+          setPageSizeCookie={setPageSizeCookie}
```

Comment:
There should be just one `setPageSize` handler which uses the cookie state. There shouldn't be the need to have an additional state variable for it.